src/System.Data.SqlClient/src/System/Data/SqlClient/SNI/SSRP.cs

```diff
@@ -34,7 +34,7 @@ internal static int GetPortByInstanceName(string browserHostName, string instanc
             }
             catch (SocketException se)
             {
-                throw new Exception(SQLMessage.SqlServerBrowserNotAccessible(), se);
+                throw new System.InvalidOperationException(SQLMessage.SqlServerBrowserNotAccessible(), se);
```

Comment:
generally one has `using System;` and do not prefix with System

Reply:
yes sorry